helix-term/src/ui/picker.rs

```diff
@@ -694,6 +694,14 @@ impl<T: 'static + Send + Sync, D: 'static + Send + Sync> Picker<T, D> {
                 cell.set_symbol(borders.horizontal).set_style(sep_style);
             }
         }
+        let header_style = cx.editor.theme.get("ui.picker.header");
+        if self.columns.len() > 1 {
+            for x in inner.left()..inner.right() {
+                if let Some(cell) = surface.get_mut(x, inner.y + 2) {
+                    cell.set_symbol(" ").set_style(header_style);
+                }
+            }
+        }
```

Comment:
Instead of manually painting the cells you can set a style for the row with `Row::style` - you can set that in the block marked with `// -- Header` below - and the table widget will take care of applying the style to the header area.

```rust
let header = Row::new(self.columns.iter().map(...)).style(header_style);
table = table.header(header);
```